cmd/clusterctl/client/cluster/mover.go

```diff
@@ -595,6 +610,49 @@ func setClusterClassPause(proxy Proxy, clusterclasses []*node, pause bool, dryRu
 	return nil
 }
 
+func waitReadyForMove(proxy Proxy, nodes []*node, dryRun bool, backoff wait.Backoff) error {
```

Comment:
It would be great if we can collect wait for move during discovery, then we can make this a no-op for most of the object, and implement a better UX for objects that are actually blocking move by printing "Wating for ...  to unblock move" before starting to wait

Reply:
Yeah that seems better. I can take a look at that as a follow-up.